graph.py

```diff
@@ -107,24 +108,29 @@ def remove_edge_with_vevlb(self, vevlb):
         return self
 
     def display(self):
-        print 't # {}'.format(self.gid)
+        display_str=''
+        print('t # {}'.format(self.gid))

```

Comment:
I don't familiar with "print_function" in both Py2 and Py3, I avoid it by using ".format()"